docker/go-carbon/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.11
+
+RUN mkdir -p /data/graphite/whisper/
+COPY ./docker/go-carbon/*.conf /etc/
+
+RUN go get github.com/lomik/go-carbon
+RUN go install github.com/lomik/go-carbon
+
+EXPOSE 2003 2004 7002 7007 2003/udp
```

Comment:
I think we can leave only the port for TCP line protocol for input. I.e. leave 2003 and 7007.